docs/Installing.asciidoc

```diff
@@ -356,3 +356,35 @@ KVM doesn't work
 * make sure the 'geekotest' user can access +/dev/kvm+
 * make sure you are not already running other hypervisors such as VirtualBox
 * when running inside a vm make sure nested virtualization is enabled (pass nested=1 to your kvm module)
+
+Tests try to access openqa via 'localhost' inside qemu which fails
+==================================================================
+
+You can use the environement variable +OPENQA_HOST+ from your perl test
+scripts, so they can communicate with the real host. This variable will
```

Comment:
well, if the information is wrong we should remove it then. I just moved it from Pitfalls.asciidoc